### DISCUSSION TITLE
Show the date of the videos in the Up Next list

### DISCUSSION THREAD

**Discussion by bjoern-vh** [FreeTubeApp/FreeTube#5238] [General]
Hello.

Thank you for this great piece of software.

I would be happy to see the date when the video was published in the Up Next sidebar.
Maybe as an optional setting. Although enough room to display would be there and even if not needed it does not annoy someone.

What do you think?

Kind regards
Björn

**Comment by absidue** [comment]
FreeTube already shows it for the local API, we can't show it when you have Invidious selected as your preferred backend though as Invidious doesn't provide that information.

**Reply by bjoern-vh** [reply]
Thanks for your reply. 
That does the trick. I didn't know :-)
